### DISCUSSION TITLE
Explorer position and size won't save

### DISCUSSION THREAD

**Discussion by LarsErikOsterud** [valinet/ExplorerPatcher#1643] [Q&A]
I'm not really sure if this is an ExplorerPatcher issue or a Windows 11 issue.
But nomatter what I do Explorer windows reopen at the same all to the right a bit down position.
Not saving either size, position, or fullsize.

If this IS a Windows 11 thing (that the problem is there) is there anything you can do in ExplorerPatcher to fix it?
(I seem to have read somewhare that StartAllBack has done somthing to fix this issue, but I like ExplorerPathcer :-)

Windows version:  22H2 22621-1992   ExplorerPacther version:  22621.1555.55.2

This MS discussion talks about a similar problem in StartAllBack, but that a newer version did fix that. So maybe you can too?
https://answers.microsoft.com/en-us/windows/forum/all/windows-11-file-explorer-windows-wont-remember/f597b6e3-fd15-4021-96d6-598191a4e31b

**Comment by pyrates999** [comment]
If StartAllBack fixes it, you should use that.

**Comment by skymaster7** [comment]
I have the same issue.

After installing ExplorerPatcher, my explorer windows no longer remember if I resize or move them. They always appear in the same position and size as when I first installed ExplorerPatcher.

To reproduce: Before installing ExplorerPatcher, I can move and resize explorer windows and it remembers the next time I open an Explorer window. As soon as I install ExplorerPatcher, it only ever opens in the same exact position and size as when I installed it.

If I uninstall ExplorerPatcher, I can resize the last open explorer window, close it, re-open explore, and it remembers the size and position of what I last used.

Windows: Version 23H2 OS build 22631.3296

ExplorerPatcher Version: 22621.3296.64.1

**Comment by clutchmaster420** [comment]
WOW this is still an issue even after a year! this is a deal breaker
